sdks/java/core/src/main/java/org/apache/beam/sdk/options/ProxyInvocationHandler.java

```diff
@@ -552,7 +557,19 @@ public void serialize(PipelineOptions value, JsonGenerator jgen, SerializerProvi
         jgen.writeObject(serializableOptions);
 
         List<Map<String, Object>> serializedDisplayData = Lists.newArrayList();
-        for (DisplayData.Item<?> item : DisplayData.from(value).items()) {
+        DisplayData displayData;
+
+        try {
+          displayData = DisplayData.from(value);
+        } catch (Exception e) {
+          String msg = "Exception thrown while collecting display data for PipelineOptions. "
```

Comment:
This is for a specific value, right? Should this include which option was a problem?
If not, is there any way to isolate this so only options that were problems actually report problems?